build_ssht_cffi.py

```diff
@@ -1,147 +1,84 @@
-import os
 import glob
+import os
 import shutil
+import subprocess
+import tempfile
 
-import yaml
 from cffi import FFI
 
 ffi = FFI()
+this_dir = ""  # os.path.dirname(os.path.abspath(__file__))
```

Comment:
We should remove the comment if we're not getting the path this way anymore.

Reply:
Agreed. removed.